server/routes/initialAppointment/handlers/initialMeetingTime.ts

```diff
@@ -17,6 +21,7 @@ export default class InitialMeetingTimeRoutes {
       appointmentTimeType,
       releaseIsOnBankHolidayOrWeekend: licence.isEligibleForEarlyRelease,
       skipUrl: this.getNextPage(licence.id.toString(), licence.typeCode, req),
+      canSkip: this.userType === UserType.PROBATION && !licence.appointmentTimeType,
```

Comment:
Added a check of `!licence.appointmentTimeType` rather than checking `appointmentTime` directly, as in if the user selected one of the non-time-specific options (eg to report immediately), `appointmentTime` will be null but the question should still be non-skippable.

It seems like there was a migration run to backfill the `appointmentTimeType` for previously-entered dates, so I don't think this should pose an issue for licences that existed before `appointmentTimeType` was added

Reply:
test?!